cdap-docs/reference-manual/source/http-restful-api/metrics.rst

```diff
@@ -156,8 +156,17 @@ The time range of a metric query can be specified in various ways:
    * - ``start=1385625600&`` ``end=1385629200``
      - From ``Thu, 28 Nov 2013 08:00:00 GMT`` to ``Thu, 28 Nov 2013 09:00:00 GMT``,
        both given as since the start of the Epoch
-   * - ``start=1385625600&`` ``count=3600``
-     - The same as before, but with the count given as a number of seconds
+   * - ``start=1385625600&`` ``count=3600&`` ``resolution=1s``
+     - The same as before, the count given as a number of seconds
+   * - ``start=1385625600&`` ``end=1385629200&`` ``resolution=1m``
+     - From ``Thu, 28 Nov 2013 08:00:00 GMT`` to ``Thu, 28 Nov 2013 09:00:00 GMT``,
+       with 1 minute resolution, will return 61 data points with metrics aggregated for each minute.
+   * - ``start=1385625600&`` ``end=1385632800&`` ``resolution=1h``
+     - From ``Thu, 28 Nov 2013 08:00:00 GMT`` to ``Thu, 28 Nov 2013 10:00:00 GMT``,
+       with 1 hour resolution, will return 3 data points with metrics aggregated for each hour.
+
+**Note:**
+``resolution=auto`` will auto determine time resolution based on the time difference calculated from start and end time.
```

Comment:
Can you explain a bit better what "auto determine time" means? What algorithm is followed to calculate the time difference?